db/main/structure.sql

```diff
@@ -3,13 +3,12 @@ SET lock_timeout = 0;
 SET idle_in_transaction_session_timeout = 0;
 SET client_encoding = 'UTF8';
 SET standard_conforming_strings = on;
-SELECT pg_catalog.set_config('search_path', '', false);
```

Comment:
@igorwwwwwwwwwwwwwwwwwwww I ran `script/dump-schema-docker.sh` but I still get this diff, of changes my new migration should not have caused. Any ideas?

Reply:
@aakritigupta did the script run successfully? you need to have docker installed and running for it to work. when I run it locally, i get the diff without the extra changes.

Reply:
@igorwwwwwwwwwwwwwwwwwwww Yeah it ran successfully and I had docker running (of course :) )
Joe has the same problem here https://github.com/travis-ci/travis-migrations/pull/125

Reply:
hm the only other thing i can think of is running rspec after running `dump-schema-docker.sh` (overwrites the file again), or that you have a custom `config/database.yml`.

Reply:
FYI #126 was merged – I hope that helps here :)